docs/references/subcommands/snippets/commit.md

```diff
@@ -0,0 +1,43 @@
+## `fossa snippets commit`
+
+This subcommand commits the analysis performed in the `analyze` subcommand into a `fossa-deps` file.
+For more information on possible options, run `millhone commit --help`.
```

Comment:
Would you have to dump or compile `millhone` to run this command? If so, are there instructions on how to do this that we can link to here?

Reply:
Whoops, that was meant to be changed to `fossa snippets`, updated.